core/dune

```diff
@@ -0,0 +1,6 @@
+(library
+ (name generics_core)
+ (public_name generics-core)
+ (wrapped false)
```

Comment:
Apparently `wrapped false` should only be used for old libraries. But otherwise you'd have to refer to this library as `Generics_core.Core` which seems a bit obtuse. (Ideally you could refer to it as `Generics.Core`, but that doesn't work))